pydoll/browser/page.py

```diff
@@ -72,6 +78,16 @@ def dom_events_enabled(self) -> bool:
             bool: True if DOM events are enabled, False otherwise.
         """
         return self._dom_events_enabled
+    
+    @property
+    def intercept_file_chooser_dialog(self) -> bool:
```

Comment:
```suggestion
    def intercept_file_chooser_dialog_enabled(self) -> bool:
```

just to maintain the same pattern :)